CSS/index.css

```diff
@@ -0,0 +1,197 @@
+body {
+    margin: 0;
+    background-color: rgb(50, 50, 50);
+    color: white;
+    font-family: "Playfair Display", Verdana, Geneva, Tahoma, sans-serif;
+}
+
+h1 {
+    text-align: center;
+    color: #eb8d46;
+}
+
+h2 {
+    color: #eb8d46;
+}
+
+main {
+    padding: 120px 20px;
+}
+
+a {
+    color: #3f5db7;
+    font-style: italic;
+}
+
+section {
+    padding: 10px 20px;
+    background-color: rgb(50, 50, 50);
+    border: black solid 5px;
+    margin: 20px 5px;
+
+}
+
+.imgMain {
+    display: block;
+    margin-top: 20px;
+    margin-left: auto;
+    margin-right: auto;
+    width: 500px;
+    border: 20px solid #3f5db7;
+}
+
+/* Tablet */
+
+@media (max-width:1024px) {
```

Comment:
Was a little hard to see what was changing, so I actually tried your query with different border colors - and it does work properly.

Reply:
My goal was to make it look similar 😅. Sorry if it was difficult to see the changes!